ccp/modules/exporter-compose.yml

```diff
@@ -0,0 +1,67 @@
+services:
+  exporter:
+    image: docker.verbis.dkfz.de/ccp/dktk-exporter:latest
+    container_name: bridgehead-ccp-exporter
+    environment:
+      LOG_LEVEL: "INFO"
+      EXPORTER_API_KEY: "${EXPORTER_API_KEY}"
+      CROSS_ORIGINS: "https://${HOST}"
+      EXPORTER_DB_USER: "exporter"
+      EXPORTER_DB_PASSWORD: "${EXPORTER_DB_PASSWORD}" # Set in exporter-setup.sh
+      EXPORTER_DB_URL: "jdbc:postgresql://exporter-db:5432/exporter"
+      CLEAN_TEMP_FILES_CRON_EXPRESSION: "0 0 1 * * *"
+      TEMP_FILES_LIFETIME_IN_DAYS: "1"
+      CLEAN_WRITE_FILES_CRON_EXPRESSION: "0 0 2 * * *"
+      WRITE_FILES_LIFETIME_IN_DAYS: "30"
```

Comment:
This does not hurt per se but consider putting these default values into the container.